### ISSUE TITLE
Why so few images in train.txt

### ISSUE THREAD

**Issue by soulslicer** [rbgirshick/py-faster-rcnn#654]
Hi all,

I am rather confused. Looking at the voc datasets, or even other datasets online..the Layout folder has test, train and val.txt. I assume the test and validation set to be small, but the training set listed seems to be a very small number as well.

Also, there is a main folder, with test, train and val for each class too with -1 and 0. I'm not sure why it's like this.

I ask because I want to create my own multilabel dataset, and I am not sure how to setup these txt files properly
